data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/s3/S3FileReaderTest.java

```diff
@@ -26,12 +24,13 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
 
 @ExtendWith(MockitoExtension.class)
 class S3FileReaderTest {
+    private static final Long CONTENT_LENGTH_EXCEEDING_THRESHOLD = 8_000_000L;
+    private static final Long CONTENT_LENGTH_SMALLER_THAN_THRESHOLD = 1_000_000L;
```

Comment:
same comment as above. Also, it probably makes sense to define ONE_MB as 1L << 20` and use it in all these places.